apps/src/lib/util/audioApi.js

```diff
@@ -70,6 +71,7 @@ export const commands = {
       }
       studioApp.cdoSounds.playURL(url, {
         volume: 1.0,
+        loop: !!opts.loop,
```

Comment:
This is the line where the magic actually happens - the core sound system already had this capability, we just weren't exposing the parameter.